src/lib/modules/Document/DocumentListEntry/DocumentListEntry.js

```diff
@@ -102,6 +102,13 @@ class DocumentListEntry extends Component {
     return image;
   };
 
+  renderSubtitle = (metadata) => {
+    const subtitle = !_isEmpty(metadata.alternative_titles)
+      ? metadata.alternative_titles.find((e) => e.type === 'SUBTITLE')
+      : null;
+    return subtitle ? subtitle.value : null;
+  };
```

Comment:
use find function of lodash, much cleaner
```suggestion
  renderSubtitle = (metadata) => {
    const subtitle = find(metadata.alternative_titles, { 'type': 'SUBTITLE' });
    return subtitle?.value;
  };
```